services/model.js

```diff
@@ -520,7 +540,7 @@
                 that.set(request.length > 1 ? request[0] : attribute, target)
               }
               if (angular.isUndefined(item)) {
-                that.set(attribute, !target)
+                that.set(attribute, target)
```

Comment:
The toggle feature is supposed to flip the boolean value, so removing this would cause it to do nothing.